src/App.js

```diff
@@ -64,22 +37,24 @@ class App extends Component {
     }
   }
 
-  displayMovieDetails = () => {
-    if (this.state.selectedMovie.poster_path === '') {
-      return <AllMovieContainer movies={this.state.allMoviesData} selectMovie={this.selectMovie}/>
-    } else {
-      return <MovieDetailContainer selectedDetails={this.state.selectedMovie}/>
-    }
-  }
-
   render() {
     return (
       <main className='main'>
         <nav className='nav-bar'>
-          <button onClick={this.returnHome} className='home-button'>🍿Rancid Tomatillos</button>
+          <Link to='/' className='home-button'>🍿Rancid Tomatillos</Link>
         </nav>
-        {this.state.error && <h2>Sorry, there was a problem with our network</h2>}
-        {!this.state.error && this.displayMovieDetails()}
+        <Route exact path='/' render={() => {
```

Comment:
Adding the fetch calls into the Routes is smart, hopefully this is something we are actually allowed to do!